src/components/FruitCounter.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+
+export function FruitCounter({fruitName}) {
+    const [fruit, setFruit] = React.useState(0)
+    return (
+        <div className="counter" >
+            <label> {fruitName} </label>
+            <button type="button" id={fruitName} disabled={fruit < 1 && true} onClick={() => setFruit(fruit - 1)}>-
```

Comment:
```suggestion
            <button type="button" id={fruitName} disabled={fruit < 1} onClick={() => setFruit(fruit - 1)}>-
```
Dit was wat overbodig 😉